.github/workflows/ci.yml

```diff
@@ -26,11 +27,11 @@ jobs:
       - name: Install Build Dependencies
         run: |
           sudo apt-get update
-          sudo apt-get -qq install texlive-latex-extra texlive-fonts-recommended tex-gyre texlive-lang-german latexmk
+          sudo apt-get -qq install python3-poetry texlive-latex-extra texlive-fonts-recommended tex-gyre texlive-lang-german latexmk
       - name: Install Python Dependencies
-        run: pip install -r requirements.txt
+        run: poetry install --no-dev
```

Comment:
Good point. However, the version of `poetry` that comes from `apt` in Ubuntu 22.04 is 1.1.12 and doesn't provide ``--without``, yet. I suggest to stay with the deprecated version as long as we rely on the older version in CI. Particularly, since we usually won't need this option in our dev environments.